### PR TITLE
Try music-metadata?

### PR DESCRIPTION
**Pull request by Borewit**
Would you like to give [music-metadata](https://github.com/borewit/music-metadata) a try?

It will work with any audio file around, promises build in, simple API, lot's of metadata you can access.

Please let me know if you need any help.

This PR:
1. Use music-metadata to retrieve metadata from audio tracks
2. Use path to to extract filename from path, more OS independent (in Windows the slashes are the ~~wrong~~ other way around)

### CODE REVIEW

File: src/js/View.js [hunk 1]
```diff
@@ -166,21 +167,16 @@ module.exports = class View {
...
-              artist.innerText = meta.artist ? meta.artist : 'Unknown';
-            },
-            onError: err => console.error(err)
-          });
+        self._musicMetadata.parseFile(files[i].path).then(metadata => {
+
+          const songName = self._path.basename(files[i].path);
+          const common = metadata.common;
+
+          name.innerText = common.track ? common.title : songName;
```
Comment (Borewit) [inline src/js/View.js]: Should be `common.title`

**Comment by CodeF0x** [comment]
Hi, thanks for suggesting this change! As it seems, migrating from jsmediatags to music-metadata would be a fix for #101 (testing locally with your fork, the in issue #101 mentioned stuttering doesn't appear), so I would be very happy to try out music-metadata. 

But also when testing, I noticed that [updating the album cover](https://github.com/CodeF0x/violin/blob/master/src/js/View.js#L325) is not migrated to music-metadata yet. Could you maybe migrate that part too, please? Then I'd be happy to merge your request!

**Comment by Borewit** [comment]
I had the changes already done, but forgot to include the `View.js` in my commit.

**Comment by CodeF0x** [comment]
Ah okay, I see. Then I will merge tomorrow, thanks!

**Comment by CodeF0x** [comment]
Thanks for your contribution!

**Pull request merged**